src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -1012,6 +1019,7 @@ protected void consume(PacketInfo packetInfo)
                 else
                 {
                     sctpManager.handleIncomingSctp(packetInfo);
+                    ByteBufferPool.returnBuffer(packetInfo.getPacket().buffer);
```

Comment:
`usrsctp` copies all buffers into a buffer of its own (synchronously down this call chain), so we have to return here.  This is another places where it'd be nice if we could get the return "closer" to the SCTP level, as it's usrsctp-specific.